### DISCUSSION TITLE
SOLVED - Documentation on how to get started into Moccacino?

### DISCUSSION THREAD

**Discussion by wanwalk** [mocaccinoOS/mocaccino#47] [General] [answered]
Where can I find documentation on how to get started into Moccacino?  

I used Sabayon for more than a year and it was my favorite distro, but I am unable to understand most of what  I find about MocaccinoOS.

**Comment by mendsec** [comment, marked as answer]
[Click here](https://www.mocaccino.org/docs/) and go to official MocaccinoOS documentation.

**Reply by wanwalk** [reply]
Thanks for your interest. I think I have read the documentation quite a few times, but have had trouble making sense  out of it. Perhaps I am too naive, being used to more traditional (less-advanced) Linux  distros.

Reading it again, upon  your recommendation. which sounded to me like I had really missed something I wondered what would  be a correct interpretation for the following quote:

>  "A User, should be able then to install KDE by calling luet install layers/plasma and nothing else. That package should bring all the necessary components to make the “app” work as expected. The user shouldn’t be exposed to the typical OS architecture of bringing with it dozens of dependencies."

Does this mean I don't have to worry about installing a kernel, profiles and other basic stuff, as luet will take care of it as I request apps ????    

I had been looking for instructions to "install an OS" before starting installing apps. Now I am wondering if that was my mistake, as I never recognized  any instructions that I thought would for example provide a kernel.

Again I am wondering if I start by booting the iso and running
 ` luet install layers/plasma` 
 will result in the installation of everything needed to run plasma and therefore in the installation of kernel and other "basic  stuff".  

In that case would I have to be chrooted ?
Pardon my ignorance, but I really miss Sabayon  and I  am looking forwards to be able to  try Mocaccino OS.

**Reply by wanwalk** [reply]
OUCH !  Pardon me!
I have just booted a new iso  and noticed an "install" icon on the Mate desktop.  I had in my previous attempt, quite a while ago - in an older machine - not succeeded in getting to a desktop.  And I never saw any refrence to some installer in the documentation.  I guess my fantasy flew wild due to Mocaccino's uniqueness.  :-)
